Gemeinsamer Bibliotheksverbund ISBN.js

```diff
@@ -0,0 +1,238 @@
+				}];
+			}
+			//DDC is not the callNumber in Germany
+			item.callNumber = "";
+			item.complete();
+		});
+		translator.translate();
+
+	});
+}/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "import",
+		"input": "{ \"ISBN\" : \"978-3-642-00230-4\" }",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Eigentumsverfassung und Finanzkrise",
+				"creators": [
+					{
+						"firstName": "Otto",
+						"lastName": "Depenheuer",
+						"creatorType": "editor"
+					},
+					{
+						"lastName": "Symposion. Deutsche Stiftung Eigentum",
+						"fieldMode": true
+					}
+				],
+				"date": "2009",
+				"ISBN": "9783642002304",
+				"language": "ger",
+				"place": "Berlin ;Heidelberg",
+				"publisher": "Springer",
+				"series": "Bibliothek des Eigentums, Im Auftrag der Deutschen Stiftung Eigentum",
+				"seriesNumber": "7",
+				"attachments": [],
+				"tags": [
+					"Aufsatzsammlung",
+					"Aufsatzsammlung",
+					"Constitutional law",
+					"Constitutional law",
+					"Constitutional law",
+					"Eigentum",
+					"Finanzkrise",
+					"Haftung",
+					"Law",
+					"Law",
+					"Law",
+					"Online-Publikation",
+					"Online-Publikation",
+					"Ordnungspolitik"
+				],
+				"notes": [
+					{
+						"note": "\"Unter dem Thema 'Eigentumsverfassung und Finanzkrise' veranstaltete die Deutsche Stiftung Eigentum am 22. April 2009 in Berlin ein Symposion"
+					},
+					{
+						"note": "Die weltweite Finanzkrise ist Anlass, an Funktion und Wirkweise des privaten Eigentums in einer freiheitlichen Gesellschafts- und Wirtschaftsordnung zu erinnern. Privates Eigentum muss es geben, damit Verantwortung zugerechnet und Haftung realisiert, Gewinn und Verlust einem konkreten Verantwortungsträger persönlich zugerechnet werden können. Die Verletzung dieser konstitutiven Regeln einer auf privatem Eigentum basierenden Wirtschaftsordnung ist wesentlich ursächlich für das eingetretene Desaster auf den Finanzmärkten. Wie alle kulturellen Errungenschaften muss auch die Idee des privaten Eigentums, insbesondere die ihr immanente Bereitschaft zur Übernahme persönlicher Verantwortung des Eigentümers, jeder Generation erneut wieder in Erinnerung gerufen, überzeugend um sie geworben und vor allem vorbildhaft von den Akteuren in Politik und Wirtschaft vorgelebt werden. Nur so kann strukturelles Vertrauen in das Finanzsystem wieder gewonnen werden. Denn in ihrer vertrauensbildenden Kraft liegt die ordnungspolitische Funktion der Gewährleistung privaten Eigentums"
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "import",
+		"input": "{ \"ISBN\" : \"3-86688-240-8\" }",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Bilinguale Lexik nicht materieller lexikalischer Transfer als Folge der aktuellen russisch-deutschen Zweisprachigkeit",
+				"creators": [
+					{
+						"firstName": "Katrin",
+						"lastName": "Karl",
+						"creatorType": "author"
+					}
+				],
+				"date": "2012",
+				"ISBN": "9783866882416",
```

Comment:
<blockquote>So I'm not really clear on what the meaning of multiple ISBNs is. Is it different formats of publication? Different publishers? Wouldn't the metadata be different for different ISBNs then?</blockquote>

There are on the one site the publisher who can decide how they want to use their ISBN numbers. It is quite frequently that the hardcover and softcover becomes two different ISBNs. Moreover, the electronic edition becomes sometimes also another ISBN. There are other cases where the PDF edition, ePub edition, electronic edition from another aggregator can also become a seperate ISBN. On the other side, there are cataloguing rules from the libraries. There are rules about when to make one entry and when to make two entries of the same "work" but different "edition".

In our case the requested ISBN corresponds to the electronic version: http://digital.kubon-sagner.com/verlag-otto-sagner/title/7015E/ (we should extract that link as well for electronic versions). However, the other ISBN is also present in the cataloguing entry (for example cataloguing rules dictates you to take your main information from the title page which might be pdf page 3 and its backside).